### PR TITLE
Update installation scripts

### PR DESCRIPTION
**Pull request by amos-kibet**
https://3.basecamp.com/5023736/buckets/38393000/todos/7680613475

- update Erlang, Elixir, Phoenix, and PostgreSQL
- remove unnecessary dependencies
- use existing shell (bash or zsh) instead of forcing usage of zsh
- refactor scripts

### CODE REVIEW

**Comment by amos-kibet** [comment]
@almirsarajcic, I am yet to fix the failing issues on CI (they do not fail locally), but would love a review for the parts that do not necessarily depend on CI passing.

File: .mise.toml [hunk 1]
```diff
@@ -1,2 +1,2 @@
 [env]
-_.file = '.env'
+_.file = '.env.sample'
```
Comment (almirsarajcic) [inline .mise.toml:2]: That's a sample file. During onboarding, you're supposed to copy its contents into a `.env` file, as we do in all our projects. Please revert.

File: .tool-versions [hunk 2]
```diff
@@ -1,3 +1,3 @@
-elixir 1.17.1-otp-27
-erlang 27.0
+elixir 1.17.2-otp-27
```
Comment (almirsarajcic) [inline .tool-versions:1]: If you're going to change the versions in this place, you need to do the same in CI, Dockerfile and everywhere else they are referenced. Either revert or fix the version in these other places too.

File: priv/static/macOS.sh [hunk 3]
```diff
@@ -21,12 +21,29 @@ white='\033[0;37m'
 green='\033[0;32m'
 cyan='\033[0;36m'
 
+# possible shells (cat /etc/shells)
```
Comment (almirsarajcic) [inline priv/static/macOS.sh:24]: I guess it's still WIP.

File: priv/static/macOS.sh [hunk 4]
```diff
@@ -68,7 +73,7 @@ function already_installed() {
 
 function install() {
     case $1 in
-    "Xcode Command Line Tools")
+    "xcode")
```
Comment (almirsarajcic) [inline priv/static/macOS.sh:76]: It's not the whole Xcode that we're installing here.

**Review by almirsarajcic — CHANGES_REQUESTED** [review]

**Review by almirsarajcic — APPROVED** [review]

**Pull request merged**